src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
 // write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', (e) => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
+  }, 3000);
+});
+
+function showMessage(message, className = []) {
+  const messageContainer = document.createElement('div');
+
+  if (className.length > 0) {
+    for (const cl of className) {
+      messageContainer.classList.add(cl);
+    }
+  }
+
+  messageContainer.textContent = message;
+  document.body.appendChild(messageContainer);
+}
```

Comment:
I'd recommend you to separate error and success messages to make your code clean
```
const showSuccessMessage = () => {
  body.insertAdjacentHTML('beforeend', `
    <div class="message">Promise was resolved!</div>
  `);
};
const showErrorMessage = () => {
  body.insertAdjacentHTML('beforeend', `
    <div class="error-message">Promise was rejected!</div>
  `);
};
```
or even you can use one function and pass as arguments type and message
```
const showMessage = (message, type) => {
  const className = (type === 'error') ? 'error-message' : 'message';
  const html = `<div class="${className}">${message}</div>`;
  body.insertAdjacentHTML('beforeend', html);
};

showMessage('Promise was resolved!', 'success');
showMessage('Promise was rejected!', 'error');
```